### DISCUSSION TITLE
Unable to ssh after upgrade: Identity file not accessible: No such file or directory.

### DISCUSSION THREAD

**Discussion by AndreiSolodinTR** [PowerShell/Win32-OpenSSH#2184] [Q&A] [answered]
Hello, I was previously using OpenSSH_8.4p1, OpenSSL 1.1.1h  22 Sep 2020, and recently upgraded to OpenSSH_for_Windows_9.4p1, LibreSSL 3.7.3. Now with the same configuration, I am unable to ssh to a host which was working before. Here's the full log that has the debug output: https://pastebin.com/1AjC3H5t  (note that the ssh command is executed by our company python tool). One thing I noticed in the log is:

Warning: Identity file 'C:/Users/XXXXXXX/.ssh/andrei.solodin_key' not accessible: No such file or directory.

However, the file is there and accessible. In the pmon log and see that ssh.exe is trying to access

`C:\Users\XXXXXXX\'C:\Users\XXXXXXX\.ssh\andrei.solodin_key'`

which is clearly wrong, but I don't quite understand where the issue is. Is this a problem with the script or with ssh.exe?

Thanks

**Comment by AndreiSolodinTR** [comment, marked as answer]
It seems like the single quotes around the -i key_filename parameter are causing the issue. I guess the question is whether single quotes around an argument have special meaning. I presume that the quotes are there in case the key_filename path has spaces in it. I am not very familiar with windows conventions regarding handing of single quotes, or is it something specific to OpenSHH?

**Comment by maertendMSFT** [comment]
The single vs double quotes should not make any difference for Win32-OpenSSH. Without knowing the python wrapper, additional debugging is difficult. Since you found the solution we are closing this discussion, please re-open it if you have further questions.
